src/functionaltests/java/com/ericsson/ei/notifications/ttl/TestTTLSteps.java

```diff
@@ -0,0 +1,232 @@
+package com.ericsson.ei.notifications.ttl;
+
+import com.ericsson.ei.mongodbhandler.MongoDBHandler;
+import com.ericsson.ei.subscriptionhandler.InformSubscription;
+import com.ericsson.ei.utils.FunctionalTestBase;
+import com.ericsson.ei.utils.TestContextInitializer;
+import com.fasterxml.jackson.databind.ObjectMapper;
+import com.mongodb.BasicDBObject;
+import cucumber.api.java.After;
+import cucumber.api.java.Before;
+import cucumber.api.java.en.Given;
+import cucumber.api.java.en.Then;
+import cucumber.api.java.en.When;
+import org.apache.commons.io.FileUtils;
+import org.json.JSONArray;
+import org.json.JSONException;
+import org.junit.Ignore;
+import org.mockserver.client.server.MockServerClient;
+import org.mockserver.integration.ClientAndServer;
+import org.mockserver.model.Format;
+import org.mockserver.model.HttpResponse;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.beans.factory.annotation.Value;
+import org.springframework.test.context.ContextConfiguration;
+import org.springframework.test.context.TestPropertySource;
+import org.springframework.util.SocketUtils;
+
+import java.io.File;
+import java.io.IOException;
+import java.text.DateFormat;
+import java.text.SimpleDateFormat;
+import java.time.LocalTime;
+import java.util.Date;
+import java.util.List;
+import java.util.concurrent.TimeUnit;
+
+import static org.junit.Assert.assertEquals;
+import static org.mockserver.model.HttpRequest.request;
+
+@Ignore
+@TestPropertySource(properties = { "notification.ttl.value:1", "aggregated.collection.ttlValue:1",
+        "notification.failAttempt:1" })
+@ContextConfiguration(initializers = TestContextInitializer.class)
+public class TestTTLSteps extends FunctionalTestBase {
+        private static final Logger LOGGER = LoggerFactory.getLogger(TestTTLSteps.class);
+        private static final String BASE_URL = "localhost";
+        private static final String ENDPOINT = "/missed_notification";
+
+        private static final String SUBSCRIPTION_FILE_PATH = "src/functionaltests/resources/SubscriptionObject.json";
+        private static final String AGGREGATED_OBJECT_FILE_PATH = "src/functionaltests/resources/AggregatedObject.json";
+        private static final String MISSED_NOTIFICATION_FILE_PATH = "src/functionaltests/resources/MissedNotification.json";
+
+        private static String subscriptionObject;
+        private static String aggregatedObject;
+        private static String missedNotification;
```

Comment:
missedNotification is only used in one method and should maybe be a local variable in that method.